.github/workflows/python-test.yml

```diff
@@ -34,8 +34,11 @@ jobs:
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 chaos_genius --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 chaos_genius --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        # exit-zero treats all errors as warnings.
+        flake8 chaos_genius --count --exit-zero --statistics
```

Comment:
why did we remove `--max-complexity` ? I read a little about McCabe complexity threshold https://www.geeksforgeeks.org/cyclomatic-complexity/ , seems useful. alsio read this https://www.cqse.eu/en/news/blog/mccabe-cyclomatic-complexity/ which concludes that it might not be very useful and using it correct might require some thoughtfulness

Reply:
Thanks for the explanation!

I have seen flake8 mark some large functions as complex that were a good target for some refactoring. In that sense, it can be a good indicator of when a function needs refactoring. If it's a false positive, it can be ignored easily with a `# noqa: ...`.

I removed the `--max-complexity` flag from here since it's already included in our repo's `.flake8` config - https://github.com/chaos-genius/chaos_genius/blob/c780e3f00663711d4a2168cc89f40ef82f6cbb54/.flake8#L9

We should use that as a single source of config for flake8 instead of manually passing params in CI

Reply:
If that's ok, you can merge the PR

Reply:
Sorry, missed it .. closing it